gui/api/resources.py

```diff
@@ -3302,13 +3318,9 @@ def dehydrate(self, bundle):
                 )
         except Exception as err:
             bundle.data['cert_DN'] = "ERROR: " + str(err)
-            # There was an error parsing this Certificate Authority object
-            # Creating a sentinel file for the alertmod to pick it up
-            with open('/tmp/alert_invalidCA_{0}'.format(bundle.obj.cert_name),
-                      'w') as fout:
-                fout.write('')
 
-            run_alerts()
```

Comment:
An exception here doesnt necessarily mean parse failed, we should probably skip the alert from here, a lot has changed sinced.